src/main/windows/mainWindow.js

```diff
@@ -72,6 +72,7 @@ function createMainWindow(config, options) {
             contextIsolation: false,
             disableBlinkFeatures: 'Auxclick',
             spellcheck,
+            enableRemoteModule: process.env.NODE_ENV === 'test',
```

Comment:
`remote` is being deprecated in Electron 12, and removed in Electron 14. They're replacing it, so will this work with the new module?

Reply:
it is deprecated as part of `electron`, and moved towards `@electron/remote` so I hope we don't have to implement any big changes.

anyways there are already a number of complains that it requires remote, so I hope Spectron devs address that soon and most likely it will mean we'll need to expose some of the electron APIs in the preload